beacon-chain/sync/rpc_hello_test.go

```diff
@@ -42,8 +44,10 @@ func TestHelloRPCHandler_Disconnects_OnForkVersionMismatch(t *testing.T) {
 		if code == 0 {
 			t.Error("Expected a non-zero code")
 		}
-		if errMsg.ErrorMessage != errWrongForkVersion.Error() {
-			t.Errorf("Received unexpected message response in the stream: %+v", err)
+		if errMsg != errWrongForkVersion.Error() {
+			t.Log(len(errMsg))
```

Comment:
This is temporary? Do we mean to delete this later?

Reply:
yeah it could be deleted, it could stay too, it's helpful for debugging an error. Will let @rauljordan look